build-script.html

```diff
@@ -3,41 +3,49 @@
+<h3>Building the required libraries</h3>
+<p><b>1)</b> Download the Goma Library build script here: <a href="https://raw.githubusercontent.com/goma/goma/master/scripts/build-goma-dep-trilinos-12.sh">Goma Library build script</a></p>
+<p><b>2)</b> Use the script in the following way: </p>
+<pre>bash build-goma-dep-trilinos-12.sh -j[Number of Make Jobs] [Build/Install Location]</pre>
+<p>This script requires about 3GB of ram per make job and the install location should be final.</p>
+<p>As soon as everything completes, you should have the essential libraries required to install Goma!</p>
+<p><b>3)</b> Before building goma you should add the newly compiled openmpi, and SEACAS to your path.</p>
 
+<p>If you want easy access you can add these settings to your shell settings (e.g. .bashrc) otherwise you can type them in manually each time you want to run Goma.</p>
 <p>OPENMPI:</p>
 
 <pre>
-  export LD_LIBRARY_PATH="/[path to gomalibs]/openmpi-1.6.4/lib:$LD_LIBRARY_PATH"
-  export PATH="/[path to gomalibs]/openmpi-1.6.4/bin:$PATH"
+  export LD_LIBRARY_PATH="/[path to gomalibs]/openmpi-[openmpi version]/lib:$LD_LIBRARY_PATH"
+  export PATH="/[path to gomalibs]/openmpi-[openmpi version]/bin:$PATH"
 </pre>
 
 <p>SEACAS</p>
 
 <pre>
-  export PATH="/[path to gomalibs]/SEACAS-2013-12-03/bin:$PATH"  
+  export PATH="/[path to gomalibs]/SEACAS-[SEACAS version]/bin:$PATH"  
 </pre>
 
-<h3>Download Goma and Brkfix from github: <a href="https://github.com/goma">Goma on github</a></h3>
-
-<p>To build goma copy the settings.mk-example to settings.mk set the GOMA_LIBS macro to match your specified installation directory and add the makefile macro <code>FRONT_LIB =</code> to your settings.mk file.</p>
-
-<p>Goma should now be able to compile with make</p>
+<h3>Building Goma And Configuring Settings.mk</h3>
+<p><b>1)</b> Download Goma and Brkfix from github: <a href="https://github.com/goma">Goma</a>.</p>
+<p>This can be done in bash with:</p>
+<pre>git clone https://github.com/goma/brkfix && git clone https://github.com/goma/goma</pre>
+<p><b>2)</b> Copy the settings.mk-example file to "settings.mk".</p>
+<p><b>3)</b> Set GOMA_LIBS to the directory you ran the script in.</p>
+<p><b>4)</b> Change the versions of TRILINOS_TOP, SEACAS_TOP, and MPI_TOP to the versions in your library.</p>
+<p><b>5)</b> Change any other appropriate settings. For example you may need to uncomment CXXSTD = -std=gnu++11 if your compiler is recent.</p>
 
-<p>To build brkfix adjust the <code>ACCESS</code> to match your seacas install from the build script <code>[Install Location]/SEACAS-2013-12-03</code></p>
+<br>
+<p>Goma should now be able to compile with make!</p>
 
+<p>If you get any errors then it is probably something which can be fixed with settings.mk.</p>
 
+<br>
+<p>Finally, to build brkfix adjust the <code>ACCESS</code> to match your seacas install from the build script <code>[Install Location]/SEACAS-2013-12-03</code></p>
```

Comment:
This probably should be changed to match the normal build script install:

Something like change the trilinos directory in Makefile.Trilinos to the build directory and trilinos version.

Then make -f Makefile.Trilinos